packages/vuetify/src/components/VSelect/VSelect.ts

```diff
@@ -790,7 +790,8 @@ export default baseMixins.extend<options>().extend({
       window.requestAnimationFrame(() => {
         menu.getTiles()
         keyCodes.up === keyCode ? menu.prevTile() : menu.nextTile()
-        menu.activeTile && menu.activeTile.click()
```

Comment:
When using up & down keys to cycle through available select items, VSelect use menu.activeTile.click() event to do selection. However when dropdown list is not active, this MouseEvent is from clicking the position of x===0, y===0 which is equivalent to clicking outside parent Dialog or Menu. Therefore, Dialog or Menu will be closed by "clicking outside"

Solution: In order to select new item, Instead of dispatching wrong `MouseEvent` from an invisible dom, directly set select Item in `VSelect` level